lib/ui/painting/image_decoder_impeller.cc

```diff
@@ -266,6 +266,7 @@ sk_sp<DlImage> ImageDecoderImpeller::UploadTextureToPrivate(
   texture_descriptor.format = pixel_format.value();
   texture_descriptor.size = {image_info.width(), image_info.height()};
   texture_descriptor.mip_count = texture_descriptor.size.MipCount();
+  texture_descriptor.compression_type = impeller::CompressionType::kLossy;
```

Comment:
This will be really unhappy for photos, especially if they're wide gamut.

Some people might not care but we probably need a flag of some sort to control whether you get this or not, at the very least for images.

Reply:
The images I've checked look identical with this flag on/off. Do you have any particular ones in mind?

Reply:
@gaaclarke probably has some from the wide gamut work

Reply:
Flutter gallery has some photographs that are in the adobergb colorspace you can test with those.  Also it's worth verifying that it doesn't mess up the gamut, there is an integration test in `//flutter/dev/integration_tests/wide_gamut_test/`.

Usually where these compression problems come into play in games isn't complex images but really simple images like pixel art.  Unity defaults to compressed textures but you have to turn them off frequently if you are using simple images.

Reply:
We're not aware of any circumstances where this causes any loss in fidelity